doc/colvars-refman-main.tex

```diff
@@ -4660,3 +4663,37 @@ \subsection{LAMMPS keywords}
 
 }
 
+
+\cvsec{Syntax changes from older versions}{sec:colvars_config_changes}
+
+The following is a list of syntax changes in Colvars since its first release.
+Many of the older keywords are still recognized by the current code: the primary purpose of this list is to let you aware of improvements that affect old configuration files.
+
+The syntax of the latest version of Colvars can always be accessed at:\\
+\cvnamdonly{\url{https://colvars.github.io/colvars-refman-namd/colvars-refman-namd.html}}
+\cvlammpsonly{\url{https://colvars.github.io/colvars-refman-lammps/colvars-refman-lammps.html}}
+\cvvmdonly{\url{https://colvars.github.io/colvars-refman-vmd/colvars-refman-vmd.html}}
+
+If you are using any of the NAMD and VMD tutorials:\\
+\url{https://www.ks.uiuc.edu/Training/Tutorials/}\\
+please be aware that \emph{several of these tutorials have not been actively maintained}: while they are updated, the following list will help you reconcile any inconsistencies.
```

Comment:
while they are updated -> pending their update

Reply:
I already changed this part in the next commit, and made it more generic (we cannot be sure whether the tutorial under consideration will be updated).